sdks/python/apache_beam/io/gcp/gcsio.py

```diff
@@ -99,6 +99,40 @@ def get_or_create_default_gcs_bucket(options):
         bucket_name, project, location=region)
 
 
+def create_storage_client(pipeline_options, use_credentials=True):
+  """Create a GCS client for Beam via GCS Client Library.
+
+  Args:
+    pipeline_options(apache_beam.options.pipeline_options.PipelineOptions):
+      the options of the pipeline
+    use_credentials(bool): whether to retrieve credentials from pipeline
+      options or not
+
+  Returns:
+    A google.cloud.storage.client.Client instance
+  """
+  if use_credentials:
+    credentials = auth.get_service_credentials(pipeline_options)
+  else:
+    credentials = None
+
+  if credentials:
+    google_cloud_options = pipeline_options.view_as(GoogleCloudOptions)
+    from google.api_core import client_info
+    beam_client_info = client_info.ClientInfo(
+        user_agent="apache-beam/%s (GPN:Beam)" % beam_version.__version__)
+    return storage.Client(
+        credentials=credentials.get_google_auth_credentials(),
+        project=google_cloud_options.project,
+        client_info=beam_client_info,
+        extra_headers={
+            "x-goog-custom-audit-job": google_cloud_options.job_name
+            if google_cloud_options.job_name is not None else "UNKNOWN"
```

Comment:
```suggestion
            if google_cloud_options.job_name else "UNKNOWN"
```

Reply:
rationale: an empty string will not be a helpful ID ""

Reply:
Done.